api-gateway/src/main/resources/static/scripts/bill-history/bill-history.controller.js

```diff
@@ -22,12 +23,102 @@ angular.module('billHistory')
             { ownerId: '10', firstName: 'Carlos', lastName: 'Esteban' }
         ];
 
+        self.ownersUUID = [
```

Comment:
Did you try to find a way to obtain the owners without having to hardcode them?

Reply:
I did and could after several attempts I fell back on this.

Reply:
There should be a getAllOwners method, possible implementation for the future year for sure.